scripts/compress_jointcal_hsc_raw_test_data.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+"""Copy in the relevant raw files from RC/w_2020_14/DM-24359-sfm, zero out the
+imageHDUs in the raws.
```

Comment:
We only need header info for ingest so the files can all have a single pixel in the data arrays. It won't matter.

Reply:
Good to know.  This leaves things nice and small anyway.

Reply:
We do use bbox info from the header as well, and I'm not sure shrinking to one pixel would preserve that.

Reply:
In any event, I used the zeroing out method which seems to me to be the safest and certainly slims things down sufficiently.

Reply:
I've gotten errors in the past when I tried making the files have a single pixel; zeroing and compressing results in files that are still very small, but have the same shape.